Sources/KinescopeSDK/View/Player/KinescopeVideoPlayer.swift

```diff
@@ -0,0 +1,63 @@
+import AVKit
+
+public class KinescopeVideoPlayer: KinescopePlayer {
+
+    // MARK: - Private Properties
+
+    private let player: AVPlayer
```

Comment:
```suggestion
    private let player = AVPLayer()
```
и из init можно убрать

Reply:
это нужно для выбора плеера в случае цикличного видео

Reply:
точно
пока оставляем
в следующем спринте переделаем чутка